portability-types-common/src/test/java/org/datatransferproject/types/common/models/MediaObjectTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.datatransferproject.types.common.models;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class MediaObjectTest {
```

Comment:
thanks for adding more test coverage! these tests are great, but I'm just curious: is there one particular case you're seeing in practice (that inspired your test coverage)? Eg: is it empty strings coming back? or are you getting non-URI ("illegal") content set on the objects?